scripts/gdprdeleteall.php

```diff
@@ -0,0 +1,42 @@
+#!/usr/local/bin/php -q
+<?php
+/**
+ * DO NOT RUN THIS SCRIPT WILLY NILLY
+ * this script will delete the personal data of any members that have been emailed regarding deletion and have not acted
+ * you should probably run the gdpremail script first then wait a week or two
+ * 
+ * run with "php gdprdeleteall.php"
+ */
+
+use \MyRadio\Database;
+use \MyRadio\MyRadioException;
+use \MyRadio\MyRadioError;
+use \MyRadio\MyRadioEmail;
+use \MyRadio\MyRadio\CoreUtils;
+use \MyRadio\ServiceAPI\MyRadio_User;
+
+ini_set('log_errors', 1);
+ini_set('error_log', '/tmp/php-mailparser-error.log');
+ini_set('display_errors', 'On');
+
+require_once '../src/Controllers/root_cli.php';
+
+$db = Database::getInstance();
+
+$time = strtotime("-1 year", time());
+$date = date("Y-m-d", $time);
+
+$db->query(
+    'UPDATE public.member
+	SET data_removal=\'default\'
```

Comment:
default?

Reply:
This reverts users account back to default if they have signed in since the warning email